lib/utils/parseCalcExpression/parser.jison

```diff
@@ -50,6 +50,9 @@
 "("                                                   return 'LPAREN';
 ")"                                                   return 'RPAREN';
 
+#\{([\s\S]*?)\}                                       return 'UNKNOWN'; // scss variable
```

Comment:
// scss interpolation

Reply:
Thank you for pointing out!
I fixed this.